docker-compose-local.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 
 services:
   db:
-    build: ./db
```

Comment:
I think this file should be named `docker-compose-prod.yml` based on this change

Reply:
Good point that makes sense